modules/reactor/src/meshgenerators/PatternedHexPeripheralModifier.C

```diff
@@ -67,6 +67,9 @@ PatternedHexPeripheralModifier::PatternedHexPeripheralModifier(const InputParame
     _pattern_pitch_meta(declareMeshProperty<Real>("pattern_pitch_meta", 0.0)),
     // Use HexagonConcentricCircleAdaptiveBoundaryMeshGenerator for control drum meshing
     _is_control_drum_meta(declareMeshProperty<bool>("is_control_drum_meta", false)),
+    _hexagon_peripheral_trimmability(
+        declareMeshProperty<bool>("hexagon_peripheral_trimmability", false)),
+    _hexagon_center_trimmability(declareMeshProperty<bool>("hexagon_center_trimmability", false)),
```

Comment:
they look center-trimmable to me?
what's the hold up?

Reply:
This generator is to modify the peripheral region of an assembly to help stitch it with other assemblies using HexPatternMG.

We do not need to stitch (also cannot) a center-trimmed assembly with other assemblies. So, No point to do this.